slasher/service/service.go

```diff
@@ -176,6 +179,25 @@ func (s *Service) startSlasher() {
 	}()
 }
 
+func (s *Service) loadSpanMaps(err error, slasherServer rpc.Server) {
+	lt, err := slasherServer.SlasherDB.LatestIndexedAttestationsTargetEpoch()
+	if err != nil {
+		log.Errorf("Could not extract latest target epoch from indexed attestations store:%v", err)
+	}
+	for i := uint64(0); i < lt; i++ {
+		ias, err := slasherServer.SlasherDB.IndexedAttestations(i)
+		if err != nil {
+			log.Errorf("Got error while trying to retrieve indexed attestations from db:%v", err)
+		}
+		for _, ia := range ias {
+			slasherServer.UpdateSpanMaps(s.context, ia)
+		}
+		if i%5 == 0 {
+			log.Infof("update span maps for epoch: %d", i)
```

Comment:
```suggestion
			log.Infof("Update span maps for epoch: %d", i)
```

If this happens a lot, feel free to switch to debug..